src/main/java/org/datadog/jmxfetch/util/LogLevel.java

```diff
@@ -0,0 +1,114 @@
+package org.datadog.jmxfetch.util;
+
+import java.util.logging.Level;
+
+/**
+ * LogLevel used for internal logging to match Datadog Agent levels.
+ * Comparison table with java.util.logging:
+ * <pre>
+ *  JUL     |  JMXFetch LogLevel
+ * ----------------------------
+ *  ALL     |  ALL
+ *  SEVERE  |  ERROR
```

Comment:
Would be good to also mention `FATAL` here

Reply:
`FATAL` shouldn't be in this table because it's not a `LogLevel`, it's only supported as a config log level string that's then converted to `LogLevel.ERROR`, as mentioned below